pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -2033,17 +2033,34 @@ var (
 	add = func(logger klog.Logger, queue *PriorityQueue, pInfo *framework.QueuedPodInfo) {
 		queue.Add(logger, pInfo.Pod)
 	}
+	expectNoError = func(what string, err error) {
+		if err != nil {
+			panic(fmt.Sprintf("Unexpected error from %s: %v", what, err))
+		}
+	}
 	addUnschedulablePodBackToUnschedulablePods = func(logger klog.Logger, queue *PriorityQueue, pInfo *framework.QueuedPodInfo) {
```

Comment:
let's change the function name to: `popAndRequeueAsUnschedulable`. Similarly for the other one

Reply:
Done.